src/System.Private.CoreLib/src/System/Diagnostics/Eventing/RuntimeEventSourceHelper.Windows.cs

```diff
@@ -16,15 +16,18 @@ internal sealed class RuntimeEventSourceHelper
 		internal static int GetCpuUsage()
 		{
 			// Returns the current process' CPU usage as a percentage
-			long procUserTime;
-			long procKernelTime;
-			long systemUserTime;
-			long systemKernelTime;
 
 			int cpuUsage;
 			
-			Interop.Kernel32.GetProcessTimes(Interop.Kernel32.GetCurrentProcess(), out _, out _, out procKernelTime, out procUserTime);
-			Interop.Kernel32.GetSystemTimes(out _, out systemUserTime, out systemKernelTime);
+			if (!Interop.Kernel32.GetProcessTimes(Interop.Kernel32.GetCurrentProcess(), out _, out _, out long procKernelTime, out long procUserTime))
+			{
+				throw new Exception("GetCpuUsage: Could not get process time.");
```

Comment:
This may better just return 0.

(Otherwise, we want all error messages to be globalized.)

Reply:
If we throw during the callback (which this is), `IncrementingPollingCounter` will emit an event with this error message. I thought that might be better for diagnosing rather than just returning 0 (which could be a feasible value for this counter).

Reply:
FWIW, the Unix side returns 0 on OS API error as well: https://github.com/dotnet/corefx/blob/master/src/Native/Unix/System.Native/pal_time.c#L156

Reply:
The changes of these APIs failing is very low. My original feedback was really just about not returning random numbers in the unlikely case that these APIs throw.

Reply:
Ok - Will just return 0 on error for consistency :-)